packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-intervention-policy-group-form.vue

```diff
@@ -53,8 +54,9 @@
 		<div class="input-row">
 			<div class="label-and-input">
 				<label for="lower-bound">Lower bound</label>
-				<tera-input-number
+				<tera-input
 					class="p-inputtext-sm"
+					type="number"
 					:min-fraction-digits="1"
 					:max-fraction-digits="10"
```

Comment:
these aren't valid props

Reply:
You are correct. Im not sure if we want to add this functionality in the future to our input or what so i left them